h/admin/views/__init__.py

```diff
@@ -6,6 +6,7 @@ def includeme(config):
     config.include('.admins')
     config.include('.badge')
     config.include('.features')
+    config.include('.cohorts')
```

Comment:
Personally I'd be inclined to have these views in the `features.py` module, as they're really part of the same logical chunk of functionality, but this isn't a killer either way.